metals/src/main/scala/scala/meta/internal/decorations/SyntheticsDecorationProvider.scala

```diff
@@ -201,7 +201,8 @@ final class SyntheticsDecorationProvider(
   }
 
   private def areSyntheticsEnabled: Boolean = {
-    userConfig().showImplicitArguments || userConfig().showInferredType || userConfig().showImplicitConversionsAndClasses
+    val showInferredType = !userConfig().showInferredType.contains("false")
```

Comment:
```suggestion
    val showInferredType = !userConfig().showInferredType.contains("false") && userConfig().showInferredType.nonEmpty
```

Reply:
LGTM, smart usage of && trick which I didn't recognize at first glance